sweetest/src/main/java/com/mysugr/sweetest/framework/dependency/DependencyConfigurations.kt

```diff
@@ -6,7 +6,7 @@ import kotlin.reflect.KClass
 interface DependencyConfigurationConsumer {
     val all: Collection<DependencyConfiguration<*>>
     operator fun <T : Any> get(clazz: KClass<T>): DependencyConfiguration<T>
-    fun <T : Any> getAssignableTo(clazz: KClass<T>): DependencyConfiguration<T>
+    fun <T : Any> getAssignableTo(clazz: KClass<T>): DependencyConfiguration<T>?
     fun <T : Any> getAssignableFrom(clazz: KClass<T>): DependencyConfiguration<T>
```

Comment:
this function just got inconsistent with `getAssignableTo` (now nullable result), but it's not used. can you remove it?